src/pages/createAppointment/[postId].tsx

```diff
@@ -0,0 +1,118 @@
+import RightCard from "@/components/createAppointment/RightCard";
+import { User } from "@/types/User";
+import { PostInfo } from "@/types/Post";
+
+export default function Home() {
+  const router = useRouter();
+  const supabaseClient = useSupabaseClient<Database>();
+  const userStatus = useContext(userContext);
+  const [postInfo, setPostInfo] = useState<PostInfo | null>(null);
+  const [availableParticipants, setAvailableParticipants] = useState<User[] | null>(null);
+  const [selectedParticipants, setSelectedParticipants] = useState<User[]>([]);
+  const [participantCountError, setParticipantCountError] = useState<boolean>(false);
+
+  const postId = parseInt(router.query.postId as string);
+
+  function handleClickBack() {
+    router.back();
+  }
+
+  function handleParticipantsChange(participants: User[]): void {
+    setParticipantCountError(false);
+    setSelectedParticipants(participants);
+  }
+
+  function onSubmit() {
+    if (selectedParticipants.length == 0) {
+      setParticipantCountError(true);
+      return;
+    }
+
+    if (postInfo) {
+      CreateAppointment(postId, postInfo, selectedParticipants, supabaseClient)
+        .then(() => {
+          // router.push(viewAppointment);
+          return;
+        })
+        .catch((err) => {
+          console.log(err);
+          return;
+        });
+    }
+  }
+
+  useEffect(() => {
+    if (!postId || !userStatus.user) return;
+    GetPostWithParticipantsByPostId(postId, supabaseClient)
+      .then((p) => {
+        setPostInfo(p);
+        if (!p.participants) return;
+        setAvailableParticipants(p.participants)
+      })
+      .catch((err) => {
+        console.log(err);
+        return;
+      });
+  }, [supabaseClient, postId, userStatus.user]);
+
+  if (userStatus.isLoading) return <Loading />;
+  if (!userStatus.user) {
+    router.push(PagePaths.login);
+    return;
+  }
+  if (!postInfo || !availableParticipants) return <Loading />;
+  return (
+    <>
+      <Navbar />
+      {/*  ArrowBackIcon */}
+      <Box display="flex" paddingBottom="40px">
+        <Link>
+          <ArrowBackIcon
+            fontSize="large"
+            sx={{ position: "absolute", margin: "3vh 0 0 3vh", color: "black" }}
+            onClick={handleClickBack}
+          />
+        </Link>
+      </Box>
+      {/*  Title */}
+      <Box>
+        <Typography variant="h1" sx={{ fontWeight: "700", fontSize: "40px" }}>
+          Create Appointment
+        </Typography>
+      </Box>
+      {/*  Left and Right Card */}
+      <Box display="flex" justifyContent="center" padding="40px">
+        <Grid container spacing="40px" width="80vw">
+          {/*  Left Card */}
+          <Grid item xs={12} md={6} style={{ display: "flex" }}>
+            <LeftCard postInfo={postInfo} isUnClick={true} />
+          </Grid>
+          {/*  Right Card */}
+          <Grid item xs={12} md={6} style={{ display: "flex" }}>
+            <RightCard
+              images={postInfo.images}
```

Comment:
If the post contains no image, the `Image` header and the content of `alt` should not appear.
<img width="1246" alt="image" src="https://user-images.githubusercontent.com/62459518/224495600-1a0d533f-55e5-4363-a800-d8d20bc333c6.png">

Reply:
In this picture,I think it have image.but it can't fetch (it show "location" instead.)
You may try to create appointment without picture.It will disappear.